packages/rich-text-editor/src/vaadin-rich-text-editor.js

```diff
@@ -554,8 +561,8 @@ class RichTextEditor extends ElementMixin(ThemableMixin(PolymerElement)) {
   }
 
   /** @protected */
-  ready() {
-    super.ready();
+  connectedCallback() {
```

Comment:
@web-padawan - The editor is now created when connected. Not sure if this is the most efficient way to do this, but since quill adds the listeners internally, this ensures that they are there when the component is detached and then attached again.

Reply:
Generally this makes sense. I'd still keep `ready()` method with stuff unrelated to Quill.
For example, `vaadin-overlay-open` event listener can be still added in `ready()`.

Reply:
I have now moved all the stuff unrelated to Quill in `ready()`
May I request another review please. 🙇‍♂️